entry_types/scrolled/package/src/editor/config.js

```diff
@@ -18,7 +21,12 @@ editor.registerEntryType('scrolled', {
       editor
     });
   },
-  outlineView: EntryOutlineView
+  outlineView: EntryOutlineView,
+
+  isBrowserSupported() {
+    return (agent.matchesSafari() || agent.matchesChrome());
```

Comment:
Maybe we can start with just excluding mobile browsers (e.g. `!matchesMobilePlatform`). Then we can fine tune the exact matchers in a later step while still being able to continue testing the editor in all desktop browsers once this is merged.